stable/redis-ha/README.md

```diff
@@ -93,6 +93,8 @@ The following table lists the configurable parameters of the Redis chart and the
 | `exporter.port`           | Exporter port                                                                                                                                                                                            | `9121`                                                                                     |
 | `exporter.annotations`    | Prometheus scrape annotations                                                                                                                                                                            | `{prometheus.io/path: /metrics, prometheus.io/port: "9121", prometheus.io/scrape: "true"}` |
 | `exporter.extraArgs`      | Additional args for the exporter                                                                                                                                                                         | `{}`                                                                                       |
+| `exporter.script.configmap`             | Let you run a custom lua script from a configmap. The corresponding environment variable `REDIS_EXPORTER_SCRIPT` will be set automatically                                                 |                                                                                            |
+| `exporter.script.keyname`               | Name of the key inside configmap which contains your script                                                                                                                                |                                                                                            |
```

Comment:
I wonder if this is too flexible, may be better just to have a static key as you can't reference multiple scripts.